controll/controller.js

```diff
@@ -1,14 +1,20 @@
-const { app } = require("../app")
 const { fetchTopics } = require("../model/model")
+const fs = require("fs/promises")
 
 
 function getTopics(req, res, next){
     fetchTopics().then((allTopics) => {
-        res.status(200).send(allTopics)
+        res.status(200).send({topics : allTopics})
     })
 }
 
 
+function getApi(req, res, next){
```

Comment:
It's ok to use fs here, but not necessary. We could just require the endpoints.json into this file